### PR TITLE
Update Changelog

### PR DESCRIPTION
**Pull request by riedgar-ms**
We hadn't been filling in our changelog, so fill in a quick summary.

### CODE REVIEW

**Review by hildeweerts — APPROVED** [review]

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ### v0.6.1
 
+* Bugfix for `_create_group_metric_set()`. Fixes the list of metrics computed for regression
+  and adds a set of metrics for 'probability' problems
+* Updated 'Credit Card' notebook
+* Added some overlooked `MetricFrame` tests
+* Various documentation tweaks and enhancements
```
Comment (adrinjalali) [inline CHANGES.md:9]: would it be possible to start having PR numbers (or links with `:pr:number` directive) here?

Comment (riedgar-ms) [reply CHANGES.md]: Done. Although I can't seem to figure out the short syntax. A bare `#<n>` doesn't work, nor does `:pr:<n>`

**Comment by adrinjalali** [comment]
Do we have the right sphinx roles enabled @riedgar-ms ? https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html

**Comment by riedgar-ms** [comment]
> Do we have the right sphinx roles enabled @riedgar-ms ? https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html

But surely that won't affect the rendering of CHANGES.md on GitHub?

**Review by adrinjalali — APPROVED** [review]
:rofl: fair enough. I somehow thought our changelog is going to be on the website and it's a part of our rendered documentation (which I think is a good idea in general). But for this PR then I'm happy with what we have, we can think about moving the changelog to the website in another issue/pr.

**Comment by riedgar-ms** [comment]
No objection from me to moving the Changelog into the main doc website @adrinjalali . Just for right now, it's only available on GitHub. I shall file an issue :-)

**Pull request merged**